source/blog/2017-05-10-Cornerstone-this-June.html.erb

```diff
@@ -0,0 +1,28 @@
+---
+title: "Cornerstone this June"
+date: 2017-05-10
+author: Cornerstone Management Resourse Systems
+location: CARNEGIE, Pa.
+category: News
+tags: Cornerstone, Carl Thorensen, Joe Thorensen, Kathy Ireland
+---
+<p>This June Cornerstone is proud to be participating in several events:</p>
+<p><strong>The Life Science Trainers and Educators Network's (LTEN) 46th Annual Conference.</strong><br>
+in Nashville, TN, from June 5-8, 2017.</p>
+<br>
+<a href="http://www.l-ten.org/">
+  <img src="/images/lten-logo.png" alt="LTEN" style="padding: 1em;">
+</a> 
+
+<br>
+<br>
+
+<p>As well as the,<br>
+<strong>Medical Affairs Professional Society (MAPS) 2017 Annual Meeting.</strong><br>
+in National Harbr, MD, from June 20-22, 2017.</p>
+<br>
+<a href="http://www.medicalaffairs.org/annual-meeting/">
```

Comment:
Should like to Speakers page http://www.medicalaffairs.org/annual-meeting/speakers/ But their site is borked 🖕